content/docs/command-reference/remove.md

```diff
@@ -47,61 +50,127 @@ how it helps replace data that is tracked by DVC.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Example: remove a .dvc file
+## Example: Remove stage outputs
 
-Let's imagine we have `foo.csv` and `bar.csv` files, that are already
-[tracked](/doc/command-reference/add) by DVC:
+Let's imagine we have a `train` stage in `dvc.yaml`, and corresponding files in
+the <abbr>workspace</abbr>:
+
+```yaml
+train:
+  cmd: python train.py data.csv
+  deps:
+    - data.csv
+    - train.py
+  outs:
+    - logs
+    - model.h5
+```
 
 ```dvc
 $ ls
-bar.csv  bar.csv.dvc  foo.csv  foo.csv.dvc
+dvc.lock  dvc.yaml  data.csv  data.csv.dvc  model.h5  logs  train.py
+
 $ cat .gitignore
-/foo.csv
-/bar.csv
+/data.csv
+/model.h5
+/logs
 ```
 
-This removes `foo.csv.dvc` and double checks that its entry is gone from
-`.gitignore`:
+Using `dvc remove` on the stage name will remove the stage from `dvc.yaml`, and
+corresponding entries from `.gitignore`. With the `--outs` option, the actual
+files and directories are deleted too (`logs/` and `model.h5` in this example):
 
 ```dvc
-$ dvc remove foo.csv.dvc
+$ dvc remove train --outs
 
 $ ls
-bar.csv  bar.csv.dvc  foo.csv
+dvc.lock  dvc.yaml  data.csv  data.csv.dvc  train.py
+
 $ cat .gitignore
-/bar.csv
+/data.csv
 ```
 
-> The same procedure applies to tracked directories.
+> Notice that the dependencies (`data.csv` and `train.py`) are not deleted.
 
-## Example: remove a stage and its output
+## Example: remove a specific stage output
```

Comment:
Not sure we need this example AND the next one ("remove specific data" from .dvc files)